selection.go

```diff
@@ -23,13 +21,13 @@ func (s Selector) BestRecord(k key.Key, recs [][]byte) (int, error) {
 
 	parts := strings.Split((string(k)), "/")
 	if len(parts) < 3 {
-		log.Infof("Record key does not have selectorfunc: %s", k)
+		logging.Logger(packageName).Infof("Record key does not have selectorfunc: %s", k)
```

Comment:
you can just do `log.Infof`. You've declared the logger as a global in validation.go